alcs-frontend/src/app/features/board/board.component.ts

```diff
@@ -222,12 +238,14 @@ export class BoardComponent implements OnInit, OnDestroy {
       const sorted = [];
       sorted.push(
         // high priority
+        ...mappedNoticeOfIntents.filter((a) => a.highPriority).sort((a, b) => b.activeDays! - a.activeDays!),
```

Comment:
I thought that NOIs are always at the top no matter the priority and then applications and the rest type of cards

Reply:
Not 100% sure, but with Ansley away we can fix this later if its wrong.